sympy/functions/combinatorial/factorials.py

```diff
@@ -1139,20 +1139,21 @@ class multinomial(CombinatorialFunction):
 
     """
     def __new__(self, *args):
+        if not args:
+            raise ValueError("A miniumum of 1 arg required")
         if len(args) == 1: # Multinomial(n) of any n is 1, this is Computationally Easier
             return S.One
         else:
+            args = [i for i in map(sympify, args) if not (i.is_zero)]
+            if not args:
+                args = [S.Zero]
             return super(multinomial, self).__new__(
-                self, *[i for i in map(sympify, args) if not (i.is_symbol and i.is_zero)])
+                self, *args)
 
     @classmethod
     def eval(self, *args):
-        if not args:
-            raise ValueError("A miniumum of 1 arg required")
-        args = [i for i in map(sympify, args) if not i.is_zero]
-        if not args:
-            return S.One
-        elif any(x.is_integer and x.is_negative for x in args):
+        args = [i for i in map(sympify, args)]
```

Comment:
sympification is now done by `__new__`, isn't it?

Reply:
yupp ... will remove this then.